src/sys/wait.rs

```diff
@@ -145,27 +145,34 @@ mod status {
           target_os = "netbsd"))]
 mod status {
     use sys::signal::Signal;
+    use libc::c_int;
```

Comment:
Sure, if you want to be pedantically correct.  But it's got nothing to do with DragonFly.  It you're going to change those types, please do it in a separate commit, not comingled with DragonFly specific changes.